plenario_client/client.py

```diff
@@ -0,0 +1,17 @@
+import config
+import requests
+from plenario_response.response import PlenarioResponse
+
+
+class PlenarioClient(object):
+
+    def __init__(self, api_url=None):
```

Comment:
`None` defaults with a swapped in value is poor form. Make the default value of the parameter the value in the config. I would also bust it out as a constant:

```python
DEFAULT_API_URL = config.API_URL

class PlenarioClient:
    def __init__(self, api_url: str=DEFAULT_API_URL):
```